core/templates/list.h

```diff
@@ -564,6 +564,7 @@ class List {
 	}
 
 	void move_to_back(Element *p_I) {
+		ERR_FAIL_COND_MSG(p_I == nullptr, "The element is null");
```

Comment:
This should probably be added to all the methods like it if any, but I think we can assume that the pointer is valid generally otherwise

Reply:
It was crashing because null was passed here, that's why I decided to add this to make sure it will not crash but at least give some information to the user.

I agree it that it would be better to have consistency and I would be happy to add it, as it would eliminate more possible crashes like the one I experienced. Yet, I am not sure if you suggest to add it in other places or remove here, in the end. What would be better in your opinion?

Reply:
I think to remove it and just ensure to use it safely, possibly adding it to all in a separate PR